solr/prometheus-exporter/build.gradle

```diff
@@ -61,6 +61,7 @@ task run(type: JavaExec) {
   main = project.ext.mainClass
   classpath = sourceSets.main.runtimeClasspath
   systemProperties = ["log4j.configurationFile":"file:conf/log4j2.xml"]
+  args = ["-f", "conf/solr-exporter-config.xml"]
```

Comment:
Ahh we could also add this to the classpath above like we do in the bin script.

Though it's not something we need to do in this PR.